Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1269,22 +1300,45 @@ def get_remote_data_command(service: client.Service, args: dict,
         demisto.debug("owner field was found, changing according to mapping.")
         updated_notable["owner"] = mapper.get_xsoar_user_by_splunk(
             updated_notable.get("owner")) if mapper.should_map else updated_notable.get("owner")
+    if updated_notable.get('comment'):
+        comment_entries = []
+        comments = argToList(updated_notable.get('comment'))
+        reviewers = argToList(updated_notable.get('reviewer'))
+        review_times = argToList(updated_notable.get('review_time'))
+        for comment, reviewer, review_time in zip(comments, reviewers, review_times):
+            comment_entries.append({
+                'Comment': comment,
+                'Comment time': review_time,
+                'Reviwer': reviewer
+            })
+            if (COMMENT_MIRRORED_FROM_XSOAR not in comment and last_update_splunk_timestamp
+                    and last_update_splunk_timestamp < float(review_time)):
+                # Creating a note
+                entries.append({
```

Comment:
Entry is created only in case of mirroring ?
in fetch - it's look like no entry is created...

Reply:
yes its only in mirroring case.